### DISCUSSION TITLE
Developer MeetUp?

### DISCUSSION THREAD

**Discussion by adam-grant-hendry** [pyvista/pyvista#2661] [Ideas]
I'd love to meetup with developers on this project! It's lonely behind a terminal sometimes.

Is anyone interested? It be great to poll the community and see if and when we could do it.

**Comment by adam-grant-hendry** [comment]
I moved this to a poll [Meetup? #2662 ](https://github.com/pyvista/pyvista/discussions/2662), so please vote there!

**Comment by tkoyama010** [comment]
We may be able to organize Meetings like the [simpeg project](https://simpeg.xyz/).
https://www.youtube.com/c/GeosciXyz/videos
@adam-grant-hendry 
If you want to do it online, I can help.
This meeting might be helpful to free up time for development and tutorial creation.

Also, we will have a [sprint at Scipy2022](https://www.scipy2022.scipy.org/sprints).
Please consider joining us there.

**Reply by tkoyama010** [reply]
As duplicate with https://github.com/pyvista/pyvista/discussions/2662 . I will lock this thread.
